scala-runner

```diff
@@ -27,7 +27,7 @@ addCoursier () {
   coursier_args+=("$1")
 }
 getScalaNightly() {
-  . <(curl https://raw.githubusercontent.com/scala/community-builds/"$1"/nightly.properties 2>/dev/null)
+  local nightly=$(curl https://raw.githubusercontent.com/scala/community-builds/"$1"/nightly.properties 2>/dev/null | sed -n 's/nightly=\(.*\)/\1/p')
   echo "$nightly"
```

Comment:
```suggestion
  curl https://raw.githubusercontent.com/scala/community-builds/"$1"/nightly.properties 2>/dev/null | sed -n 's/nightly=\(.*\)/\1/p'
```

No need to set and echo, no?

I think when I was researching how to do this I read about "from Bash 4".  So maybe it's a bashism, maybe it's a bash 4+ thing.  Either way, happy to switch to sed.

Reply:
right :-)